<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maditor::View::MainWindow</class>
 <widget class="QMainWindow" name="Maditor::View::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maditor</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="centralWidget">
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="gui">
    <attribute name="title">
     <string>GUI</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="guiHierarchy">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="scene">
    <attribute name="title">
     <string>Scene</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="sceneHierarchy">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="scriptEditorTab">
    <attribute name="title">
     <string>Scripting</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="Maditor::View::Editors::ScriptEditorTabWidget" name="scriptEditorTabWidget">
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gameTab">
    <attribute name="title">
     <string>Game</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QStackedWidget" name="game">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>197</red>
             <green>197</green>
             <blue>197</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>177</red>
             <green>177</green>
             <blue>177</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>197</red>
             <green>197</green>
             <blue>197</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>177</red>
             <green>177</green>
             <blue>177</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>177</red>
             <green>177</green>
             <blue>177</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>177</red>
             <green>177</green>
             <blue>177</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="notLoaded">
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="notLoadedLabel">
           <property name="text">
            <string>No Project Loaded!</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="notStarted">
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QLabel" name="notStartedLabel">
           <property name="text">
            <string>Project not Started!</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecentProjects">
     <property name="title">
      <string>Recent Projects</string>
     </property>
    </widget>
    <addaction name="actionNewProject"/>
    <addaction name="actionLoadProject"/>
    <addaction name="separator"/>
    <addaction name="menuRecentProjects"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>Editor</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuApp">
    <property name="title">
     <string>App</string>
    </property>
    <addaction name="actionInit"/>
    <addaction name="actionFinalize"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionPause"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEditor"/>
   <addaction name="menuApp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="LogsDockWidget">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Logs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="LogsDockWidgetContent">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="Maditor::View::LogsTabWidget" name="LogsWidget">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ProjectDockWidget">
   <property name="windowTitle">
    <string>Project</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="ProjectDockWidgetContent">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="Maditor::View::TreeView" name="ProjectWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ResourcesDockWidget">
   <property name="windowTitle">
    <string>Resources</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="ResourcesDockWidgetContent">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="Maditor::View::TreeView" name="ResourcesWidget">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject"/>
   <addaction name="actionLoadProject"/>
   <addaction name="separator"/>
   <addaction name="actionInit"/>
   <addaction name="actionFinalize"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionPause"/>
   <addaction name="separator"/>
   <addaction name="actionOpenVS"/>
   <addaction name="separator"/>
   <addaction name="actionTest"/>
  </widget>
  <widget class="QDockWidget" name="PerformanceDockWidget">
   <property name="windowTitle">
    <string>Performance</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="PerformanceDockWidgetContent">
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="0" column="0">
      <widget class="Maditor::View::TreeView" name="PerformanceWidget">
       <property name="locale">
        <locale language="English" country="Germany"/>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="OgreSceneDockWidget">
   <property name="windowTitle">
    <string>OgreScene</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="OgreSceneDockWidgetContent">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <widget class="Maditor::View::TreeView" name="OgreSceneWidget">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ObjectsDockWidget">
   <property name="windowTitle">
    <string>Objects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="ObjectsDockWidgetContent">
    <layout class="QGridLayout" name="gridLayout_12">
     <item row="0" column="0">
      <widget class="QTableView" name="ObjectsWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionLoadProject">
   <property name="text">
    <string>Load Project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/start.jpg</normaloff>:/icons/icons/start.jpg</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/pause.png</normaloff>:/icons/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/Stop.jpg</normaloff>:/icons/icons/Stop.jpg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionOpenVS">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/VS.png</normaloff>:/icons/icons/VS.png</iconset>
   </property>
   <property name="text">
    <string>VS</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionInit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Init</string>
   </property>
  </action>
  <action name="actionFinalize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Finalize</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Maditor::View::LogsTabWidget</class>
   <extends>QTabWidget</extends>
   <header>view/OgreWidgets/logstabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Maditor::View::Editors::ScriptEditorTabWidget</class>
   <extends>QTabWidget</extends>
   <header>view/Editors/scripteditortabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Maditor::View::TreeView</class>
   <extends>QTreeView</extends>
   <header>view/treeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionStart</sender>
   <signal>triggered()</signal>
   <receiver>Maditor::View::MainWindow</receiver>
   <slot>startApp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop</sender>
   <signal>triggered()</signal>
   <receiver>Maditor::View::MainWindow</receiver>
   <slot>stopApp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPause</sender>
   <signal>triggered()</signal>
   <receiver>Maditor::View::MainWindow</receiver>
   <slot>pauseApp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>Maditor::View::MainWindow</receiver>
   <slot>openSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInit</sender>
   <signal>triggered()</signal>
   <receiver>Maditor::View::MainWindow</receiver>
   <slot>initApp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFinalize</sender>
   <signal>triggered()</signal>
   <receiver>Maditor::View::MainWindow</receiver>
   <slot>finalizeApp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTest</sender>
   <signal>triggered()</signal>
   <receiver>Maditor::View::MainWindow</receiver>
   <slot>test()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newProject()</slot>
  <slot>startApp()</slot>
  <slot>stopApp()</slot>
  <slot>pauseApp()</slot>
  <slot>openSettings()</slot>
  <slot>initApp()</slot>
  <slot>finalizeApp()</slot>
  <slot>test()</slot>
 </slots>
</ui>
